prompt2model/dataset_generator/prompt_based.py

```diff
@@ -81,29 +77,22 @@ def __init__(
             cache_root: The root directory for caching generated examples.
 
         Raises:
-            ValueError: If an API key is not provided and set as an environment
-            variable, or if the 'max_api_calls' value is not greater than 0.
+            ValueError: If the 'max_api_calls' value is not greater than 0.
 
         Note:
-            For the OpenAI GPT-3.5 API, Temperature ranges from 0 to 2. Higher
+            Temperature ranges from 0 to 2. Higher
```

Comment:
I just want to ensure that every model that `lightllm` supports can change the temperature.

Reply:
Seems so.